webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/50-reload-policy-file-server

```diff
@@ -11,4 +11,12 @@
 
 # Signal to cf-serverd that it shall look-up the new promises
 
+ACL_BACK="/var/rudder/tmp/cf-serverd.backup"
+ACL_FILE="/var/rudder/cfengine-community/inputs/common/1.0/cf-serverd.cf"
+if [ -f "${ACL_BACK}" ]; then
+  if [ "$(stat --format=%Y "${ACL_BACK}")" -ge "$(stat --format=%Y "${ACL_FILE}")" ]; then
```

Comment:
I think we should reload if file is different, regardless of the date.

Reply:
In backup/restore cases with older date or time change on the machine it would be very hard to debug

Reply:
diff is not everywhere by default

Reply:
if the date is older and the file is different, the autdetected reload by cf-serverd shoud catchup within 60s